sass/components/_feature.scss

```diff
@@ -0,0 +1,71 @@
+.feature {
+    display: grid;
```

Comment:
I completely forgot about using `display: grid` when I was trying to fix it. It's perfect for this.